data/datacatalogue3/stagingCohorts/molgenis.csv

```diff
@@ -166,8 +166,8 @@ Repeated variables,All variables,,,,,,,,,,,Definition of a repeated sourceVariab
 Repeated variables,,is repeat of,ref,,TRUE,,Variables,resource,,,,reference to the definition of the sourceVariable that is being repeated
 Dataset mappings,,source,ref,1,TRUE,,Extended resources,,,,,datasource being mapped from
 Dataset mappings,,source dataset,ref,1,TRUE,,Datasets,source,,,,name of the table being mapped from
-Dataset mappings,,target,ref,1,TRUE,DataCatalogue,Extended resources,,,,,"model being mapped to, i.e. toModel.resource + toModel.version"
-Dataset mappings,,target dataset,ref,1,TRUE,DataCatalogue,Datasets,target,,,,name of the table being mapped to
+Dataset mappings,,target,ref,1,TRUE,catalogue,Extended resources,,,,,"model being mapped to, i.e. toModel.resource + toModel.version"
```

Comment:
why this renaming?

Reply:
we merged the data-catalogue and data-catalogue-staging. the schema on data-catalogue is called 'catalogue'

Reply:
okay. I think this is not a problem but Connor made a automatic 'demo' feature in the previews also called 'catalogue'. So we might at some point make an error where one overwrites the other involuntarily. However, we can also revert this reasoning and say the demo is more like the real deal.

Reply:
However, wat we then must do is change the 'installer' scripts. Because these assume a different name, see https://github.com/molgenis/molgenis-emx2/blob/810e6ac259eb0d92defb14f8a67802eb42a2de9b/backend/molgenis-emx2-datamodels/src/main/java/org/molgenis/emx2/datamodels/DataCatalogueCohortStagingLoader3.java#L22